clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/PingMessage.java

```diff
@@ -3,12 +3,23 @@
 
 package com.microsoft.aspnet.signalr;
 
-class PingMessage extends HubMessage {
+class PingMessage extends HubMessage
+{
+    private static PingMessage instance;
 
-    int type = HubMessageType.PING.value;
+    private PingMessage()
+    {
+    }
+
+    public static PingMessage getInstance() {
+        if (instance == null) {
```

Comment:
This doesn't need to be lazy-initialized

Reply:
Yeah, skip the lazy-init and just initialize it in the field.